pages/threads.go

```diff
@@ -44,12 +44,8 @@ func addFile(path string) (string, error) {
 		use = parts[len(parts)-1]
 	}
 
-	mdir := &pb.MobilePreparedFiles{
```

Comment:
Here's where you'll likely want to check my logic @andrewxhill and maybe @sanderpick. It doesn't seem like we need the actual preparedfiles struct... and also that is no longer exposed anyway.